### PR TITLE
indian food night quotes :D

### PR DESCRIPTION
**Pull request by ysim**
teeheehee

### CODE REVIEW

**Comment by marvinpinto** [comment]
:smile:

File: quotes.md [hunk 1]
```diff
@@ -909,3 +909,25 @@ sh# /usr/bin/sshd
...
+
+---
+
+[same situation as before]  
+
+Meg, handing YQ a greasy paper bag of popadum with a giant tear down the middle: Here, do you want some of the crispy things to snack on?  
+
+YQ: Oh, thank you!  
+
+YQ, holding the paper bag carefully: Maybe I can put these in a bowl or something.
```
Comment (des4maisons) [inline quotes.md]: I recall it slightly differently:

> [Same situation as before]
> YQ is sitting with her feet up on the couch, looking comfortable. Meg is standing up behind the couch.
> ...
> YQ stretches out her arm to accept them: Oh, thank you!
> YQ, after a pause, continuing to hold the paper bag carefully at arms length: "Maybe I should put these in a bowl or something."
...

**Comment by des4maisons** [comment]
:sheep: :sheep: :sheep: :sheep: :sheep: :sheep: baaa

**Comment by ysim** [comment]
@des4maisons updated :P

**Comment by des4maisons** [comment]
You know you don't have to listen to me right @ysim ?

**Comment by des4maisons** [comment]
Also you can merge without waiting for a comment :)

**Comment by ysim** [comment]
@des4maisons yes i know, but you were right this time ;) and ok!

**Pull request merged**